setup.py

```diff
@@ -1,25 +1,25 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-"""Installation script for `metaopt.algo.skopt.bayes`."""
+"""Installation script for `orion.algo.skopt.bayes`."""
 from setuptools import setup
 
 setup_args = dict(
-    name='metaopt.algo.skopt.bayes',
+    name='orion.algo.skopt.bayes',
```

Comment:
Shall we keep the convention that the PyPI package name is going to be the same as the Python package itself? So `orion.algo.skopt`?